test/util/edu-1990e-helpers.js

```diff
@@ -0,0 +1,22 @@
+const mock = require('./mock-helpers');
+
+// Create API routes
+function initApplicationSubmitMock() {
```

Comment:
Yes! This is actually the first refactoring I tried when I first poked around here, but I ran into a host of other issues with my local e2e test. Just put it back in, and using it for the 1995 and 1990e, but not the 1990, since the 1990 has its on special version of the shared helpers we have in `test/util/edu-helpers.js` which we can keep until we write the 1990 to use rjsf